Problem1

```diff
@@ -0,0 +1,38 @@
+class Solution {
+    public int subarraySum(int[] nums, int k) {
```

Comment:
Good Try. Please refer the class video for the optimal solution or reach out to the TAs or the professor.